schemas/core/v1/notification.json

```
@@ -28,6 +28,13 @@
           "items": {
             "type": "string"
           }
+        },
+        "emails": {
+          "description": "List of emails to direct the notification to. This won’t override notification's administrators settings. Emails list will be merged with other settings.",
+          "type": "array",
+          "items": {
+            "type": "string"
```

I think having the [format email](https://json-schema.org/understanding-json-schema/reference/string#email-addresses) will be useful. 
```suggestion
            "type": "string",
            "format": "email"
```

---

Indeed, thanks!